demo.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#set -x
+
+BLOCK_SIZE=${1:-100M} # 100Mb by default
+TMPDIR="${TMPDI:-$(mktemp -d)}"
+
+OUT="${OUT:-"${TMPDIR}/bighash"}"
+
+echo "Preparing big file at ${OUT}..."
+dd if=/dev/urandom of="$OUT" bs="$BLOCK_SIZE" count=1
+
+echo "Make 4 smaller parts from ${OUT}..."
+split -dn 4 ${OUT} ${TMPDIR}/
+
+echo -n "Big file hash: "
+TZALL=`./bin/tzsum -impl avx2 -name ${OUT} | awk '{print $1}'`
```

Comment:
Why backtick and not `$()`?
`shellcheck` usually complains.

Reply:
Agree. A blast from the past =)